assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -135,6 +142,62 @@ export const actions = {
 		const property = properties[ 0 ] || { id: PROPERTY_CREATE };
 		registry.dispatch( STORE_NAME ).selectProperty( property.id );
 	},
+
+	/**
+	 * Creates a new Analytics account.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} args              Argument params.
+	 * @param {string} args.accountName  Google Analytics account name.
+	 * @param {string} args.propertyName Google Analytics property name.
+	 * @param {string} args.profileName  Google Analytics profile name.
+	 * @param {string} args.timezone     Google Analytics timezone.
+	 * @return {Function} Generator function action.
+	 */
+	*createAccount( { accountName, propertyName, profileName, timezone } ) {
+		invariant( accountName, 'accountName is required.' );
+		invariant( propertyName, 'propertyName is required.' );
+		invariant( profileName, 'profileName is required.' );
+		invariant( timezone, 'timezone is required.' );
```

Comment:
This is somewhat inconsistent in the codebase, but I've been trying to add a bit more context to the error message otherwise it's a pain to track down where it's coming from.
E.g. "accountName is required to create an account."

Reply:
👍 improved in https://github.com/google/site-kit-wp/pull/1390/commits/407a38b75c35236bf697fe5ccd2aa20c89674866